### PR TITLE
feat(experimental): add `logsNotifications` websocket method

### PR DESCRIPTION
**Pull request by buffalojoec**

### CODE REVIEW

**Comment by buffalojoec** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1643** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1643" target="_blank">Graphite</a> 
    * **PR #1644** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1644" target="_blank">Graphite</a> 
      * **PR #1645** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1645" target="_blank">Graphite</a> 
        * **PR #1646** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1646" target="_blank">Graphite</a> 
          * **PR #1647** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1647" target="_blank">Graphite</a> 
            * **PR #1648** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1648" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1648?utm_source=stack-comment).

File: packages/rpc-core/src/rpc-subscriptions/logs-notifications.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        // null if simulation failed before the transaction was able to execute
+        // (for example due to an invalid blockhash or signature verification failure)
+        logs: readonly string[] | null;
+        // The transaction signature base58 encoded
+        signature: TransactionSignature;
+    }>
+>;
+
+// Currently, the mentions field only supports one Pubkey string per method call.
+// Listing additional addresses will result in an error.
```
Comment (steveluscher) [inline packages/rpc-core/src/rpc-subscriptions/logs-notifications.ts:21]: [infinitesunglasses](https://app.graphite.dev/macros/solana-labs)

**Review by steveluscher — APPROVED** [review]
[mess](https://app.graphite.dev/macros/solana-labs)

File: packages/rpc-core/src/rpc-subscriptions/logs-notifications.ts (hunk 1 above)
Comment (buffalojoec) [reply packages/rpc-core/src/rpc-subscriptions/logs-notifications.ts:21]: RIP: From docs

**Pull request merged**